snappy/plan.sh

```diff
@@ -0,0 +1,24 @@
+pkg_origin=core
+pkg_name=snappy
+pkg_version=1.1.0
+pkg_dirname=snappy-bde324c0169763688f35ee44630a26ad1f49eec3
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('BSD-2-Clause')
```

Comment:
Snappy appears to be released under `BSD-3-Clause`: https://github.com/google/snappy/blob/master/COPYING